src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -288,33 +286,19 @@ void updateContributions(Set<StageWithCaseData> stage) {
     StageWithCaseData getUnassignedAndActiveStageByTeamUUID(UUID teamUUID, UUID userUUID) {
         log.debug("Getting unassigned cases for user: {} in team {}", userUUID, teamUUID);
```

Comment:
Method returns a single but refers to a plural. Does the name of this method make sense at all?

Reply:
we're not even fetching cases.